upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -338,9 +338,11 @@ if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
         --data @cloud-init-global_update.json \
         https://api-gw-service-nmn.local/apis/bss/boot/v1/bootparameters
 
+    set -u
```

Comment:
I think we should write the values for KUBERNETES_VERSION and CEPH_VERSION to the myenv file after lines 303 and 304 -- so if the script exits between the two steps these variables will still resolve...

Reply:
it will be overwrite later in the global var stage. this whole export stage needs some revisit. ideally each stage should be independent, not using var from other stage. so i am digging into csi code and see how i can get the version in that export global env stage.

Reply:
csi uses regex match to find the version from file name. i will just do